src/loaders/Loader.js

```diff
@@ -121,6 +121,9 @@ THREE.Loader.prototype = {
 			};
 
 			for ( var name in m ) {
+				if ( !m.hasOwnProperty( name ) ) {
```

Comment:
this avoid inherited properties and is suggested by a few linters, i can't find a conclusive answer for this particular case other than - my lint told me to :)
Want it removed?